siac/walletcmd.go

```diff
@@ -224,8 +227,11 @@ func walletloadseedcmd() {
 }
 
 // walletloadsiagcmd loads a siag key set into the wallet.
-func walletloadsiagcmd(keyfiles string) {
-	password, err := speakeasy.Ask("Wallet password: ")
+func walletloadsiagcmd() {
+	fmt.Print("Enter Siag Location: ")
```

Comment:
More than just making it plural, perhaps also provide a short example `file1, file2, ...`